app/src/main/java/net/squanchy/notification/NotificationsIntentService.java

```diff
@@ -45,23 +49,33 @@ public void onCreate() {
     }
 
     @Override
+    public void onStart(@Nullable Intent intent, int startId) {
+        super.onStart(intent, startId);
+        subscriptions = new CompositeDisposable();
```

Comment:
this goes in `onCreate`, also I can't see the `subscriptions.clear()` in `onStop`